src/main/java/com/openlattice/datastore/services/DatastoreElasticsearchImpl.java

```diff
@@ -1252,7 +1253,14 @@ public SearchResult executeEntitySetMetadataSearch(
                 .actionGet();
 
         List<Map<String, Object>> hits = Lists.newArrayList();
-        response.getHits().forEach( hit -> hits.add( hit.getSourceAsMap() ) );
+        response.getHits().forEach( hit -> {
```

Comment:
Isn't this file deprecated in favor of the Kotlin version? Can we remove this class/file yet?

Reply:
This is actually still the one that's being used instead of the kotlin one :( I was tempted to remove it and switch over but I figured that can be a separate PR since we should probably do some extra testing there to make sure that switchover goes smoothly

Reply:
😢